core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -639,6 +639,10 @@ public enum SqlKind {
    */
   CAST,
 
+  /**
+   * The {@code SAFE_CAST} function. */
```

Comment:
It is better to add description for what is the difference between `SAFE_CAST` and `CAST`

Reply:
I'll update the comment. I did describe the function in `SqlLibraryOperators` too